cloud/datasync/src/SyncServer.ts

```diff
@@ -4,6 +4,16 @@ import {setupGlobalHandlers, setupHandlers} from './crud-handlers/HandlerMapper'
 import SyncDataSetOptions from './options/SyncDatasetOptions';
 import SyncOptions from './options/SyncGlobalOptions';
 import SyncApi from './SyncApi';
+import {BunyanLogger,LOG_LEVEL} from '../../logger-cloud/src/index';
+//import {BunyanLogger,LOG_LEVEL} from '@raincatcher/logger-cloud'; this don't work
+
+// constructor for loggers
+const logD = new BunyanLogger(LOG_LEVEL.DEBUG);
```

Comment:
+1, check this https://github.com/witmicko/raincatcher-core/blob/7f4ce88a5ce6313c1cfb4ce283dbc2ed1e60b3fd/cloud/logger-debug/src/index.ts